.github/workflows/docker.yml

```diff
@@ -13,6 +13,9 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v3
+      -
+        name: QEMU Setup
```

Comment:
Can this be moved before the checkout step so that it can be cached between builds?

Reply:
you may be right.